service/src/lib.rs

```diff
@@ -365,8 +366,12 @@ where
         self.runtime
             .block_on(launched)
             .map_err(error::CustomError::new)?;
```

Comment:
this will ofc block, therefore `fn bind` will never return, therefore `Mutex lock` acquired in `Loader.load` in `tokio::spawn_blocking` call will never get released.